environments/staging/aat.yml

```diff
@@ -115,9 +115,15 @@ cname:
   - name: "afdverify.moneyclaims"
     ttl: 3600
     record: "afdverify.hmcts-aat.azurefd.net"
+  - name: "moneyclaims"
+    ttl: 3600
+    record: "hmcts-aat.azurefd.net"
```

Comment:
Bogdan, should the name value be `"moneyclaims"` or `"cmc"` like it is here: https://github.com/hmcts/azure-platform-terraform/blob/master/environments/prod/prod.tfvars#L648-L653